google/cloud/bigquery/client.py

```diff
@@ -1183,17 +1295,22 @@ def delete_dataset(
             raise TypeError("dataset must be a Dataset or a DatasetReference")
 
         params = {}
+        path = dataset.path
         if delete_contents:
             params["deleteContents"] = "true"
 
         try:
-            self._call_api(
-                retry,
-                method="DELETE",
-                path=dataset.path,
-                query_params=params,
-                timeout=timeout,
-            )
+            span_attributes = {"path": path}
```

Comment:
Seems like we would want the value of `deleteContents` here, at least if it's set.

Reply:
will do